### PR TITLE
Added new template, fixed mtbseq template

### PR DESCRIPTION
**Pull request by GuilleGorines**
MTBSeq template was a little bit suboptimal, I think its better now!
Besides, I added the genomeev template (I swear I was told to do it, if not we can delete it later)

### CODE REVIEW

File: bu_isciii/templates/genomeev/DOC/viralrecon_sars.config
Comment (svarona) [inline bu_isciii/templates/genomeev/DOC/viralrecon_sars.config:1]: En un template de EV no tiene sentido tener un config de SARS

File: bu_isciii/templates/genomeev/DOC/viralrecon_sars_save_nohost.config
Comment (svarona) [inline bu_isciii/templates/genomeev/DOC/viralrecon_sars_save_nohost.config:1]: En un template de EV no tiene sentido tener un config de SARS

File: bu_isciii/templates/genomeev/RESULTS/README
Comment (svarona) [inline bu_isciii/templates/genomeev/RESULTS/README:1]: No hay lablog de RESULTS?

**Review by svarona — COMMENTED** [review]
Revisa los comentarios porfi

File: bu_isciii/templates/genomeev/DOC/viralrecon_sars.config
Comment (GuilleGorines) [reply bu_isciii/templates/genomeev/DOC/viralrecon_sars.config:1]: Deleted! It was a direct copy from the previous, just to have it running as soon as possible

File: bu_isciii/templates/genomeev/ANALYSIS/README [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Once the files have been placed, we have to modify the samples_ref.txt file. 
+    First column will be the exact same as the samples_id.txt file. 
+    Second column will be the name of the assemblies we downloaded in the previous step (example: RotavirusG8_GCA_002669555_1 ). Make sure that all the rows are the exact same.
+    Third column will be the name of the host (typically "human", but can be changed depending on the situation)
+
+- Execute the lablog_viralrecon. The ANALYSIS02 directory will be created and filled with the corresponding scripts. Load the modules and launch viralrecon.
+
+- Once it has ended, its time for MAG. Go to the ANALYSIS03 directory, execute the lablog, load the modules and run MAG with the specified params.
+
+- Last, but not least, go to the ANALYSIS04 directory and run the lablog, the lablog will check if during the assembly step in viralrecon, and will store the names of the samples that didnt assembly to the scaffold level in the noscaffold.txt file. Run normally the three scripts after loading the corresponding module, and that should be about everything there is to this service!
```
Comment (Shettland) [inline bu_isciii/templates/genomeev/ANALYSIS/README:25]:
```suggestion
- the lablog will check if during the assembly step in viralrecon, and will store the names of the samples that didnt assembly to the scaffold level in the noscaffold.txt file.
```
Don't know what did you mean by "the lablog will check if during the assembly step in viralrecon"

Comment (GuilleGorines) [reply bu_isciii/templates/genomeev/ANALYSIS/README:25]: Any sample assembly (performed in viralrecon) that did not reach a scaffold level (and therefore, will not be present in the assembly directory), will be noted in the noscaffold.txt. This is just for traceability and QOL mainly, no practical use whatsoever apart from quick checking

File: bu_isciii/templates/mtbseq_assembly/ANALYSIS/ANALYSIS02_MTBSEQ/lablog [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-cat ../samples_id.txt | xargs -I % echo "cd %;ls *.fastq.gz | tr '_' '\t' | cut -f 1,2 | sort -u > samples.txt;cd -" > _01_preparesamples.sh
+cat ../samples_id.txt | xargs -I % echo "mkdir %; cd %; ln -s ../../*ANALYSIS01*/01-preprocessing/trimmed_sequences/%_1.trim.fastq.gz %_lib1_R1.fastq.gz; cd -" | bash
+cat ../samples_id.txt | xargs -I % echo "cd %; ln -s ../../*ANALYSIS01*/01-preprocessing/trimmed_sequences/%_2.trim.fastq.gz %_lib1_R2.fastq.gz; cd -" | bash
+cat ../samples_id.txt | xargs -I % echo "cd %; ls *.fastq.gz | tr '_' '\t' | cut -f 1,2 | sort -u > samples.txt; cd -" | bash
 
-cat ../samples_id.txt | xargs -I @@ echo -e "srun --job-name MTBSEQ.@@  --output logs/MTBSEQ.@@.%j.log --partition middle_obx --mem 48G --chdir ${scratch_dir}/@@ --cpus-per-task 10 MTBseq --step TBfull --threads 10 --samples samples.txt &" > _02_mtbseq.sh
+cat ../samples_id.txt | xargs -I @@ echo -e "srun --job-name MTBSEQ.@@  --output logs/MTBSEQ.@@.%j.log --partition middle_idx --exclude ideafix[10-15] --mem 100G --chdir ${scratch_dir}/@@ --cpus-per-task 10 MTBseq --step TBfull --threads 10 --samples samples.txt &" > _01_mtbseq.sh
```
Comment (Shettland) [inline bu_isciii/templates/mtbseq_assembly/ANALYSIS/ANALYSIS02_MTBSEQ/lablog:10]:
```suggestion
cat ../samples_id.txt | xargs -I @@ echo -e "srun --job-name MTBSEQ.@@  --output logs/MTBSEQ.@@.%j.log --partition middle_idx  --mem 100G --chdir ${scratch_dir}/@@ --cpus-per-task 10 MTBseq --step TBfull --threads 10 --samples samples.txt &" > _01_mtbseq.sh
```
I dont think its a good idea to exclude certain nodes by default

File: bu_isciii/templates/mtbseq_assembly/ANALYSIS/ANALYSIS02_MTBSEQ/samples_all/lablog [hunk 3]
```diff
@@ -13,10 +14,10 @@ cd GATK_Bam; ln -s ../../*/GATK_Bam/* . ; cd -
 cd Mpileup; ln -s ../../*/Mpileup/* . ; cd -
 cd Position_Tables; ln -s ../../*/Position_Tables/* . ; cd -
 
-#####Create join scrpts
+##### Create join scrpts
```
Comment (Shettland) [inline bu_isciii/templates/mtbseq_assembly/ANALYSIS/ANALYSIS02_MTBSEQ/samples_all/lablog:17]:
```suggestion
##### Create join scripts
```
Just a typo

File: bu_isciii/templates/mtbseq_assembly/ANALYSIS/ANALYSIS02_MTBSEQ/lablog (hunk 2 above)
Comment (GuilleGorines) [reply bu_isciii/templates/mtbseq_assembly/ANALYSIS/ANALYSIS02_MTBSEQ/lablog:10]: The main idea was not to clog the system completely by default but sure, its easy to add anyway

File: bu_isciii/templates/genomeev/ANALYSIS/lablog_viralrecon [hunk 4]
```diff
@@ -0,0 +1,68 @@
+test_ref=$(cut -f2 samples_ref.txt | head -n1); if [ "$test_ref" == "NC_045512.2" ]; then rm create_summary_report_metagenomic.sh; mv create_summary_report_sars.sh create_summary_report.sh; else rm create_summary_report_sars.sh; mv create_summary_report_metagenomic.sh create_summary_report.sh; fi
+test_ref=$(cut -f2 samples_ref.txt | head -n1); if [ "$test_ref" == "NC_045512.2" ]; then ANALYSIS_TYPE=AMPLICONS; else ANALYSIS_TYPE=METAGENOMIC; fi
+test_ref=$(cut -f2 samples_ref.txt | head -n1); if [ "$test_ref" == "NC_045512.2" ]; then CONFIG_FILE=../../DOC/viralrecon_sars.config; else CONFIG_FILE="../../DOC/viralrecon_metagenomic.config"; fi
```
Comment (Shettland) [inline bu_isciii/templates/genomeev/ANALYSIS/lablog_viralrecon:3]: The distinction between sars and metagenomic might be unnecessary for this pipeline, but im not sure.

**Review by Shettland — APPROVED** [review]
Looks nice, good job!!

**Pull request merged**